common-tasks/backup-restore.md

```diff
@@ -48,7 +78,32 @@ Creating a Backup
 4. When you are ready, click **Next**. Qubes will proceed to create your backup. Once the progress bar has completed, you may click **Finish**.
 
 
-Restoring from a Backup
+Restoring from a Backup (R4.0 and later)
+-----------------------
+
+1. In **Qubes VM Manager**, click **System** on the menu bar, then click **Restore Qubes from backup** in the drop-down list. This brings up the **Qubes Restore VMs** window.
+
+2. Select the source location of the backup to be restored:
+
+   - If your backup is located on a [USB mass storage device](/doc/stick-mounting/), select the device in the drop-down box next to **Device**.
+   - If your backup is located in a (currently running) VM, select the VM in the drop-down box next to **AppVM**.
+
+   You must also specify the directory in which the backup resides (or a command to be executed in a VM). If you followed the instructions in the previous section, "Creating a Backup," then your backup is most likely in the location you chose as the destination in step 3. For example, if you had chosen the `~/backups` directory of a VM as your destination in step 3, you would now select the same VM and again type `backups` into the **Backup directory** field.
+
+   **Note:** After you have typed the directory location of the backup in the **Backup directory** field, click the ellipsis button `...` to the right of the field.
+
+3. There are three options you may select when restoring from a backup:
+   1.  **ignore missing templates and net VMs**: If any of the VMs in your backup depended upon a NetVM or TemplateVM that is not present in (i.e., "missing from") the current system, checking this box will ignore the fact that they are missing and restore the VMs anyway.
```

Comment:
IMO it worth noting that default netvm/template will be used instead of those missing ones.